ci/scripts/release.sh

```diff
@@ -4,6 +4,7 @@
 set -euo pipefail
 
 REPO_ROOT=${PWD}
+export GIT_SHA=${BUILDKITE_COMMIT}
```

Comment:
Previously this is not set. So it's `unknown` for pre-built binary (it's only set for docker images).

```
dev=> select version();
                  version
--------------------------------------------
 PostgreSQL 8.3-RisingWave-0.19.0 (unknown)
```